src/service/core/workflow/tests/test_helpers.py

```
@@ -640,5 +652,544 @@ def test_get_recent_tasks_default_minutes_ago_is_five(self):
+
+    def test_get_workflows_no_filters_uses_default_status_exclusion(self):
+        database = mock.Mock()
+        database.execute_fetch_command.return_value = []
+
+        result = self._run(database, order=connectors.ListOrder.ASC)
+
+        self.assertEqual(result, [])
+        args, _ = database.execute_fetch_command.call_args
+        cmd, params, return_raw = args
+        # Default (no statuses supplied) excludes FAILED_SUBMISSION.
+        self.assertIn(f'status != %s', cmd)
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)